tests/glyphdata_test.py

```diff
@@ -190,5 +193,221 @@ def test_glyphdata_no_duplicates(self):
+    "lla_uMatra-tamil": "uni0BB30BC1",
+    "lla_uuMatra-tamil": "uni0BB30BC2",
+    "llla_iMatra-tamil": "uni0BB40BBF",
+    "llla_iiMatra-tamil": "uni0BB40BC0",
+    "llla_uMatra-tamil": "uni0BB40BC1",
+    "llla_uuMatra-tamil": "uni0BB40BC2",
+    "ma_iMatra-tamil": "uni0BAE0BBF",
+    "ma_iiMatra-tamil": "uni0BAE0BC0",
+    "ma_uMatra-tamil": "uni0BAE0BC1",
+    "ma_uuMatra-tamil": "uni0BAE0BC2",
+    "mo-khmer.below.ro": "uni17D21798.ro",
+    "moMa_underscore-thai": "uni0E21005F",  # uni0E21_uni005F
+    "na_iMatra-tamil": "uni0BA80BBF",
+    "na_uMatra-tamil": "uni0BA80BC1",
+    "na_uuMatra-tamil": "uni0BA80BC2",
+    "ng_ya-deva": "uni0919094D092F",
+    "nga_uMatra-tamil": "uni0B990BC1",
+    "nga_uuMatra-tamil": "uni0B990BC2",
+    "ngoNgu_underscore-thai": "uni0E07005F",  # uni0E07_uni005F
+    "niggahita_maiCatawa-lao": "uni0ECD0ECB",
+    "niggahita_maiCatawa-lao.right": "uni0ECD0ECB.right",
+    "niggahita_maiEk-lao": "uni0ECD0EC8",
+    "niggahita_maiEk-lao.right": "uni0ECD0EC8.right",
+    "niggahita_maiTho-lao": "uni0ECD0EC9",
+    "niggahita_maiTho-lao.right": "uni0ECD0EC9.right",
+    "niggahita_maiTi-lao": "uni0ECD0ECA",
+    "niggahita_maiTi-lao.right": "uni0ECD0ECA.right",
+    "nikhahit_maiChattawa-thai": "uni0E4D0E4B",
+    "nikhahit_maiChattawa-thai.narrow": "uni0E4D0E4B.narrow",
+    "nikhahit_maiEk-thai": "uni0E4D0E48",
+    "nikhahit_maiEk-thai.narrow": "uni0E4D0E48.narrow",
+    "nikhahit_maiTho-thai": "uni0E4D0E49",
+    "nikhahit_maiTho-thai.narrow": "uni0E4D0E49.narrow",
+    "nikhahit_maiTri-thai": "uni0E4D0E4A",
+    "nikhahit_maiTri-thai.narrow": "uni0E4D0E4A.narrow",
+    "nna_uMatra-tamil": "uni0BA30BC1",
+    "nna_uuMatra-tamil": "uni0BA30BC2",
+    "nnna_uMatra-tamil": "uni0BA90BC1",
+    "nnna_uuMatra-tamil": "uni0BA90BC2",
+    "nno-khmer.below.narrow1": "uni17D2178E.narrow1",
+    "nno-khmer.below.narrow2": "uni17D2178E.narrow2",
+    "noNu_underscore-thai": "uni0E19005F",  # uni0E19_uni005F
+    "nya_iMatra-tamil": "uni0B9E0BBF",
+    "nya_uMatra-tamil": "uni0B9E0BC1",
+    "nya_uuMatra-tamil": "uni0B9E0BC2",
+    "nyo-khmer.full.below.narrow": "uni17D21789.full.below.narrow",
+    "p_ta-deva": "uni092A094D0924",
+    "pa_uMatra-tamil": "uni0BAA0BC1",
+    "pa_uuMatra-tamil": "uni0BAA0BC2",
+    "pho-khmer.below.ro": "uni17D21797.ro",
+    "po-khmer.below.ro": "uni17D21796.ro",
+    "ra_uMatra-deva": "uni09300941",
+    "ra_uMatra-tamil": "uni0BB00BC1",
+    "ra_uuMatra-deva": "uni09300942",
+    "ra_uuMatra-tamil": "uni0BB00BC2",
+    "reph_anusvara-deva": "uni0930094D0902",
+    "ro-khmer.pre.narrow": "uni17D2179A.narrow",
+    "rra_iMatra-tamil": "uni0BB10BBF",
+    "rra_iiMatra-tamil": "uni0BB10BC0",
+    "rra_uMatra-tamil": "uni0BB10BC1",
+    "rra_uuMatra-tamil": "uni0BB10BC2",
+    "sa_iMatra-tamil": "uni0BB80BBF",
+    "sa_iiMatra-tamil": "uni0BB80BC0",
+    "sa_uMatra-tamil": "uni0BB80BC1",
+    "sa_uuMatra-tamil": "uni0BB80BC2",
+    "sh_r-deva": "uni0936094D094D0930",  # uni0936094D0930094D
```

Comment:
i'm not familiar with devanagari conjucts but looks like there's some reordering that is (or is not) happening here?
/cc @schriftgestalt 

```diff
- uni0936094D094D0930
+ uni0936094D0930094D
```

Reply:
I think this is addressed by the mini Devanagri shaper in #818, so I’m not going to worry about it here.